assets/stylesheets/layout/_main.scss

```diff
@@ -275,54 +280,58 @@ html.iframed {
 $sidebar-width-max: 272px;
 $sidebar-width-min: 228px;
 
+.wp {
```

Comment:
Let's use `.layout` here as we should remove the other classes from the element eventually.